pkg/kclient/volumes.go

```diff
@@ -82,3 +111,36 @@ func (c *Client) GetPVCsFromSelector(selector string) ([]corev1.PersistentVolume
 
 	return pvcList.Items, nil
 }
+
+// generateVolumeNameFromPVC generates a random volume name based on the name
+// of the given PVC
+func generateVolumeNameFromPVC(pvc string) string {
+	return fmt.Sprintf("%v-%v-volume", pvc, util.GenerateRandomString(nameLength))
```

Comment:
Nit: Any reason why we wouldn't just want to call utils.NamespacedOpenShiftObject here?

Reply:
I think volume name is not limited to 63 limit since its not a kube resource(need to confirm tho).

Reply:
You're right, it might be 253

Reply:
actually, does not allow my vol name to be more than 63, updated